### ISSUE TITLE
Add information about job postings / other advertisements

### ISSUE THREAD

**Issue by jangondol** [pyconsk/pycon.sk#13]
As the exposure of the Slovak Python community grows, more people are **sending their job offers** (this trend will likely grow). This is sometimes desirable but *we don't want to be a free "spamming" channel* for recruiters. What to do?

Ideas from the mailing list: follow the [Rubyslava's policy](https://www.facebook.com/notes/rubyslava/politika-zverej%C5%88ovania-inzer%C3%A1tov/468207449916620) of accepting ads for either a fee (60 EUR in Rubyslava's case), in exchange for a talk at the meetup, or for another form of compensation.

The policy doesn't exist yet. Shall we just copy/adapt the one from Rubyslava and publish it on the site?

**Comment by ricco386** [comment]
I agree we should have such policy for social networks. Can you get some text together (ideal in Slovak and English language), we will publish somewhere. Not sure where at this moment, as for the pycon.sk page we dont need to put it there as I mentioned in bug #14 in future we can have whole portal for python where will be place for advertising and leave pycon page just for conference and meetups (even thou they might move to other domain as well, but this is future discussion we should have after pycon).

**Comment by jangondol** [comment]
"Členovia občianskeho združenia SPy alebo sponzori PyCon môžu bezplatne zverejniť jeden inzerát ročne. Pre ostatných zverejníme inzerát buď po zaplatení 64 EUR na účet o.z. SPy alebo bezplatne výmenou za prezentáciu na meetupe. Individuálna dohoda je taktiež možná."

Why 64 EUR? Because it looks good in binary... ;-) (We can change this, of course, it is just a proposal...)

@ricco386 Let me know if you'd like me to change / translate this.
